patches/minecraft/net/minecraft/world/level/storage/LevelStorageSource.java.patch

```diff
@@ -1,42 +1,30 @@
 --- a/net/minecraft/world/level/storage/LevelStorageSource.java
 +++ b/net/minecraft/world/level/storage/LevelStorageSource.java
-@@ -178,6 +_,10 @@
-    }
+@@ -299,6 +_,19 @@
+          return LevelStorageSource.this.m_78229_(this.f_78271_.toFile(), LevelStorageSource.m_78247_(p_78281_, p_78282_));
+       }
  
-    static BiFunction<File, DataFixer, PrimaryLevelData> m_78247_(DynamicOps<Tag> p_78248_, DataPackConfig p_78249_) {
-+       return getReader(p_78248_, p_78249_, null);
-+   }
++      public void readAdditionalLevelSaveData() {
```

Comment:
This is never called on the server, so I'm not sure how you plan on reading the data there.